pkg/cmd/server/server.go

```diff
@@ -53,6 +55,15 @@ func main() {
 	defer klog.Flush()
 	log = klogr.New()
 
+	if *cpuprofile != "" {
+		f, err := os.Create(*cpuprofile)
+		if err != nil {
+			log.Error(err, "failed to create cpu profile")
+		}
+		pprof.StartCPUProfile(f)
```

Comment:
if you got error, you cannot StartCPUProfile
you can change log.Error to log.Fatal or add os.exit(1)